src/celengine/timeline.h

```diff
@@ -38,7 +39,7 @@ class Timeline
     void markChanged();
 
 private:
-    std::vector<TimelinePhase*> phases;
+    std::vector<std::shared_ptr<TimelinePhase> > phases;
```

Comment:
in modern c++ it's not required to add a space between > and > in templates

Reply:
Good to know. :-)